Remove links for switching between AJAX and Java Applet versions of web interface.

diff a/data/remote/index.html b/data/remote/index.html
--- a/data/remote/index.html
+++ b/data/remote/index.html
@@ -31,7 +31,6 @@
 <!--<table id="bigtable">
 <tr><td>-->
 <div class="ui-layout-north">
-	<div id="modeswitch"><span class="active">AJAX version</span> / <a href="/applet.html">Applet version</a></div>
 	<img src="img/logo.png" alt="FatRat" id="logo" />
 	<div id="errors" class="ui-state-error"></div>
 	<div id="toolbar" class="ui-widget-header ui-corner-all">
